app/src/main/java/com/gmail/epamtraininghw/SimpleReceiver.kt

```diff
@@ -0,0 +1,28 @@
+package com.gmail.epamtraininghw
+
+import android.app.Activity
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import android.widget.TextView
+
+class SimpleReceiver : BroadcastReceiver() {
+    companion object {
+        private const val MESSAGE = "message"
+    }
+
+    lateinit var messageTextView: TextView
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        val activity: Activity = context as Activity
+        messageTextView = activity.findViewById(R.id.incomingMessageTextView)
+
+        if (intent!!.extras != null) {
```

Comment:
it's not related to current task, but if intent marked as nullable it's wrong say compiler that intent is not null